R/module_teal.R

```diff
@@ -199,29 +210,23 @@ srv_teal <- function(id, modules, raw_data, filter = teal_slices()) {
           datasets_singleton
         }
       }
-      datasets <- module_datasets(modules)
-
-      logger::log_trace("srv_teal@4 Raw Data transferred to FilteredData.")
-      datasets
+      module_datasets(modules)
     })
 
-    reporter <- teal.reporter::Reporter$new()
-    if (is_arg_used(modules, "reporter") && length(extract_module(modules, "teal_module_previewer")) == 0) {
-      modules <- append_module(modules, reporter_previewer_module())
-    }
-
     # Replace splash / welcome screen once data is loaded ----
     # ignoreNULL to not trigger at the beginning when data is NULL
     # just handle it once because data obtained through delayed loading should
     # usually not change afterwards
     # if restored from bookmarked state, `filter` is ignored
-    observeEvent(datasets_reactive(), ignoreNULL = TRUE, once = TRUE, {
```

Comment:
why remove `ignoreNULL` and `once` arguments?

Reply:
I'll revert, `once = TRUE`, `ignoreNULL` is TRUE by default